dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/AppsResourceTest.java

```diff
@@ -773,6 +773,92 @@ public void Test_Secret_Serializer_Returned_Values_Match_Descriptor_Verify_Hidde
         }
     }
 
+    /**
+     * Test we save a secret that is described as a Select List
+     * Given scenario: We create a secret then we update the select value sending one of the values
+     * Expected Result: Only the public value  was supposed to be updated. The other 2 hidden fields should remain the same since we sent a bunch of *****
+     * @throws IOException
+     */
+    @Test
+    public void Test_Select_List_Param()
+            throws IOException {
+       //Select list definition
+       final List<Map<String,String>> list = ImmutableList.of(
+           ImmutableMap.of("label","-","value",""),
```

Comment:
Codacy found an issue: [The String literal "label" appears 4 times in this file; the first occurrence is on line 787](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5895122)